### ISSUE TITLE
Message.reactionCounts  is null after sendMessage

### ISSUE THREAD

**Issue by maxmaxandr** [GetStream/stream-chat-android#472]
In io.getstream.chat.android.client.models.Message reactionCounts is nullable field. 
So all code related to it don't check on nullable 
but when I create a message via viewModel.sendMessage(message)
and the message is sent on the backend in ChannelControllerImpl
``` 
 val runnable = {
                val result = channelController.sendMessage(message)
                result
            }

```
We got a response from backend where reaction_counts is null
```
{
  "message": {
    "id": "9-7a74e16d-5f87-4944-9190-7f37212a8797",
    "text": "Rrr",
    ....
    "latest_reactions": [],
    "own_reactions": [],
    "reaction_counts": null,
    "reaction_scores": {},
    ...
  },
  "duration": "26.43ms"
}
```

So after deserialization, the message has reactionCounts with null.

It causes a problem in other places.  like addReaction extension when it used.

**Comment by elevenetc** [comment]
Thanks for report. Set of fields are indeed might be null under some circumstances. We're going to handle that on parser level.

**Comment by maxmaxandr** [comment]
HI, any plans when it will be fixed.

**Comment by elevenetc** [comment]
@maxmaxandr fixed in `4.2.8-beta-1`.
